apps/demo/app/demo/[component].tsx

```diff
@@ -200,15 +200,17 @@ const ComponentDemo = ({config}: {config: DemoConfiguration}) => {
           rounding="lg"
         >
           {Object.keys(propValues).map((prop) => (
-            <Field
-              key={prop}
-              label={config.demoOptions?.controls?.[prop]?.label ?? startCase(prop)}
-              {...config.demoOptions?.controls?.[prop]}
-              value={propValues[prop]}
-              onChange={(value: any) => {
-                setPropValues({...cloneDeep(propValues), [prop]: value});
-              }}
-            />
+            <Box key={prop} paddingY={2}>
+              <Field
+                // TODO: change label -> title once field props are updated
```

Comment:
Let's start adding these todos to Notion here: https://www.notion.so/flourishhealth/03129716a7974bc3ba9151a187c6c717?v=d9e661b10fce4c5bbf7372b891e53220

Reply:
good call